bridge/package.json

```diff
@@ -101,4 +102,4 @@
     "css-what": "~5.0.1",
     "glob-parent": "~5.1.2"
   }
-}
+}
```

Comment:
Please, add always a new line at the end of the file. The same for all files in this PR. :)

Reply:
The new line is added, thanks for the note!

Reply:
package-lock.json is updated too.